lib/Voice/gateway.ex

```diff
@@ -30,7 +30,7 @@ defmodule Alchemy.Voice.Gateway do
 
     def heartbeat do
       now = DateTime.utc_now() |> DateTime.to_unix
-      build_payload(now * 1000, :heartbeat)
+      build_payload(now * 100, :heartbeat)
```

Comment:
I'm not sure this is necessary?

Looking at the docs, it just specifies that you need a *nonce*
https://discordapp.com/developers/docs/topics/voice-connections
We don't even need to multiply by anything, come to think of it.

If we want to guarantee uniqueness of the nonce, we could use millisecond granularity instead of the default second in DateTime